src/lib/variables/variables.ts

```diff
@@ -134,34 +133,42 @@ export const getValue = (
   assertIsVariables(result);
 
   return result;
-};
+}
 
 /**
  * Turns a variable (both primitive and non-primitive) into a string.
  */
-export const variableToString = (variable: Variables): string => {
+export function variableToString(variable: Variables): string {
   if (typeof variable === 'string') {
     return variable;
   }
 
+  if (variable === undefined) {
+    return 'undefined'; // TODO: for both undefined and null, return 'None'?
```

Comment:
opinion: It makes sense to me, but it is probably a breaking change, even though I hope nobody is stringifying undefined into url parameters.

Reply:
I thought about it as well, but when I looked how it is used, I found it in `replaceParameters` where it is concatenated to string, so the result will be the same.

But where it can cause problems are filters and http, where it is used to create headers, body, etc. So, it would send `'undefined'`, which is not desired. 

I will explicitly return `undefined`, so it is not left to knowledge of `JSON.stringify` behavior.

Reply:
But that also means `variableToString`, should be returning `string | undefined` because now it is not clear it can actually return `undefined`.

Reply:
And I was wrong in original comment because in filters and http is used `variablesToStrings`, which is removing `undefined`. And actually, it is removing `None` so even `null`. Which might be a problem. Even though previously it would return `'null'` which feels even worse.